docs/developer/concepts/duck-typing.md

```diff
@@ -129,7 +138,7 @@ spec:
     name: a-name
 ```
 
-(with indirect subject)
+With an indirect subject, Binding is expected to be in the following shape:
```

Comment:
To be consistent with the other changes below, I suggest:

```suggestion
With an indirect `subject`, Binding is expected to be in the following shape:
```

Reply:
Thanks, done so.